pkg/chartutil/load.go

```diff
@@ -48,6 +48,9 @@ func Load(name string) (*chart.Chart, error) {
 		return nil, err
 	}
 	if fi.IsDir() {
+		if validChart, err := IsChartDir(name); !validChart {
```

Comment:
It might be worth running a few `helm lint` commands with this change. I think the linter uses this function to load charts. It _might_ change the linter's behavior if it can no longer load a chart that does not have a Chart.yaml. (Previously, I think it generated a specific error, but I am not sure.)